cassovary-core/src/main/scala/com/twitter/cassovary/util/FastUtilUtils.scala

```diff
@@ -61,4 +61,6 @@ object FastUtilUtils {
 
   def newInt2IntOpenHashMap(): mutable.Map[Int, Int] =
```

Comment:
Have a look here. I'm creating fastutils collection that is only an implementation of scala's built-in collection (mutable.Map). If you use something like this, you can have 
```
private val outboundLists: mutable.IndexedSeq[mutable.IndexedSeq[Int]] = new ArrayBuffer()
```
IMO, it's more convenient to hide fastutils api and use scala's that is easier in use (ex. map, find and so on), esp. in tests.

This works using the fact that IntArrayList implements java.util.List that can be converted to scala.collection.mutable.Seq using scala.collection.JavaConversions._.